lib/flight-howto/guide.rb

```diff
@@ -118,6 +120,31 @@ def content
       parse_result.content
     end
 
+    # TODO: When merging with metadata tag work, replace this read call and
+    # remove this comment
+    def render_manpage
+      html  = Kramdown::Document.new(content,
+                                     smart_quotes: ['apos', 'apos', 'quot', 'quot'],
+                                     typographic_symbols: { hellip: '...', ndash: '--', mdash: '--' },
```

Comment:
They have been converted so they use `ASCII` instead of `Unicode`. There is no particular reason they are the same. We can change `mdash` to `---` if you prefer?

Reply:
If that works, I think it would be better.